book/common/definitions.tex

```diff
@@ -542,71 +539,72 @@
 	title={Onto}]
 		
 	Let $f:X\to Y$ be a function.
-	We say $f$ is \emph{onto} (or \emph{surjective}) if every point in the codomain of $f$ gets mapped to.
+	We say $f$ is \emph{onto}\index{Onto function}\index[definitions]{Onto function} (or \emph{surjective}) if every point in the codomain of $f$ gets mapped to.
 	That is $\Range(f)=Y$.
 \end{SaveDefinition}
 
 \begin{SaveDefinition}[
 	key=IdentityMatrix,
 	title={Identity Matrix}]
 	
-	An \emph{identity matrix} is a square matrix with ones on the diagonal
+	An \emph{identity matrix}\index{Matrix!identity}\index[definitions]{Matrix!identity} is a square matrix with ones on the diagonal
```

Comment:
`identity` -> `identity matrix`. We want to clarify that we're not talking about a "matrix identity" but the "identity matrix"